frontend/index.html

```diff
@@ -70,35 +84,9 @@
 		/>
 		<meta property="twitter:image" content="/hero.png" />
 
-		<link rel="apple-touch-icon" href="/logo192.png" />
-		<!--
-      manifest.json provides metadata used when your web app is installed on a
-      user's mobile device or desktop. See https://developers.google.com/web/fundamentals/web-app-manifest/
-    -->
-		<link rel="manifest" href="/manifest.json" />
-		<link rel="preconnect" href="https://fonts.googleapis.com" />
```

Comment:
Ended up removing this preconnect since there's a stylesheet link to the same origin 2 lines below, which would make the connection anyways. As far as I can tell, this isn't saving us any time.

The preconnect below to fonts.gstatic.com on the other hand does help because without it, we'd only start connecting after loading https://fonts.googleapis.com/css2?family=Roboto+Mono&display=swap in a waterfall.

Reply:
sweet, thanks for catching this